src/hooks/useVaultFactory.ts

```diff
@@ -1,60 +1,54 @@
 import assert from 'assert';
 
 import { ZERO_ADDRESS } from 'config/constants';
-import { getToken, knownTokens, TAssetEnum } from 'config/networks';
+import { TAssetEnum } from 'config/networks';
 import { useApeSnackbar } from 'hooks';
-import { useCurrentOrg } from 'hooks/gql';
 import { useFakeVaultApi } from 'recoilState/vaults';
 
 import { useContracts } from './useContracts';
 
 import { IVault } from 'types';
 
-export function useVaultFactory() {
+export function useVaultFactory(orgId?: number) {
```

Comment:
Adding `orgId` as an argument instead of using `useCurrentOrg` was a key step in making this hook testable. Otherwise it depends upon not only the GraphQL API but a long chain of Recoil atoms.